File: makemy_trip/makemy_trip/src/entite/reservation_guide.java
package entite;

public class reservation_guide {
	
	private int id_resGuide,id_guide, id_resVol;
	

	public reservation_guide(int id_resGuide, int id_guide, int id_resVol)
	{
		this.id_guide=id_guide; 
		this.id_resGuide=id_resGuide; 
		this.id_resVol=id_resVol; 
	}
	public reservation_guide(int id_guide, int id_resVol)
	{
		this.id_guide=id_guide; 
		this.id_resVol=id_resVol; 
	}
	

	public int getId_resGuide() {
		return id_resGuide;
	}

	public void setId_resGuide(int id_resGuide) {
		this.id_resGuide = id_resGuide;
	}

	public int getId_guide() {
		return id_guide;
	}

	public void setId_guide(int id_guide) {
		this.id_guide = id_guide;
	}

	public int getId_resVol() {
		return id_resVol;
	}

	public void setId_resVol(int id_resVol) {
		this.id_resVol = id_resVol;
	}

	@Override
	public String toString() {
		return "reservation_guide [id_resGuide=" + id_resGuide + ", id_guide=" + id_guide + ", id_resVol=" + id_resVol
				+ "]";
	}

	
	
	

}

File: src/Entity/ReservationGuide.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * ReservationGuide
 *
 * @ORM\Table(name="reservation_guide", indexes={@ORM\Index(name="id_resVol", columns={"id_resVol"}), @ORM\Index(name="id_guide", columns={"id_guide"})})
 * @ORM\Entity
 */
class ReservationGuide
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_resGuide", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idResguide;

    /**
     *
     *
     * @ORM\ManyToOne(targetEntity="GuideTouristique")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_guide", referencedColumnName="id_guide")
     * })
     */
    private $idGuide;

    /**
     *
     *
     * @ORM\ManyToOne(targetEntity="Res_Vol")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_resVol", referencedColumnName="id_resVol")
     * })
     */
    private $idResvol;

    /**
     * @return int
     */
    public function getIdResguide()
    {
        return $this->idResguide;
    }

    /**
     * @param int $idResguide
     */
    public function setIdResguide(int $idResguide): void
    {
        $this->idResguide = $idResguide;
    }

    /**
     *
     */
    public function getIdGuide()
    {
        return $this->idGuide;
    }

    /**
     *
     */
    public function setIdGuide( $idGuide): void
    {
        $this->idGuide = $idGuide;
    }

    /**
     *
     */
    public function getIdResvol()
    {
        return $this->idResvol;
    }

    /**
     *
     */
    public function setIdResvol( $idResvol): void
    {
        $this->idResvol = $idResvol;
    }


}
